queries.sql

```diff
@@ -30,3 +30,29 @@ select count(*) from animals;
 select * from animals where escape_attempts = 0;
 select avg(weight_kg) from animals;
 select avg(escape_attempts) as average_escapes from animals where date_of_birth between '1990-01-01' and '2000-01-01';
+
+select name from animals inner join owners on animals.owners_id = owners.id
+    where owners.full_name = 'Melody Pond';
+select animals.name from animals inner join species on animals.species_id = species.id
+    where species.name = 'Pokemon';
+select * from animals a inner join species s on a.species_id = s.id where s.name = 'Pokemon';
+select a.name, o.full_name from animals a right join owners o on a.owners_id = o.id;
+select a.name, o.full_name from animals a inner join owners o on a.owners_id = o.id 
+    where a.species_id = (select id from species where name = 'Digimon') and o.full_name = 'Jennifer Orwell';
+select species_id, count(*) as count from animals group by species_id;
+select name from animals inner join species on animals.species_id = species.id where species_id = owners_id;
+select name from animals inner join owners  on animals.owners_id = owners.id  where animals.escape_attempts = 0;
+
+SELECT o.full_name, COUNT(*) AS count
+FROM owners o
+JOIN animals a ON o.id = a.owner_id
+GROUP BY o.full_name
+ORDER BY count DESC
+LIMIT 1;
+
+SELECT animals.* FROM animals 
+LEFT JOIN owners ON animals.owner_id = owners.id 
+LEFT JOIN species ON animals.species_id = species.id 
+WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+
```

Comment:
- [ ] As suggested bybthe previous reviewer I would kindly suggest that you should please add the screenshot of the following queries to your Pull Request 

- List of all animals that are Pokemon (their type is Pokemon).
- List all owners and their animals, remember to include those that don't own any animal.
- List all Digimon owned by Jennifer Orwell.

Reply:
@brainconnect93 kindly check the PR description because I added it.

Reply:
I forgot to update it , Thanks @brainconnect93

Reply:
well resolved! ✔️